components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -297,29 +335,50 @@ public function front( $content ) {
 
 			// build Pods object for current item
 			global $post;
-			$pods = pods( $current_post_type, $post->ID );
-
-			if ( $this_pod['single'] && is_singular( $current_post_type ) ) {
-				// load the template
-				$content = $this->load_template( $this_pod['single'], $content, $pods, $this_pod['single_append'] );
-			} //end if
-			// check if we are on an archive of the post type
-			elseif ( $this_pod['archive'] && is_post_type_archive( $current_post_type ) ) {
-				// load the template
-				$content = $this->load_template( $this_pod['archive'], $content, $pods, $this_pod['archive_append'] );
+			$pod_name = $current_post_type;
+			$pod_item = $post->ID;
+			if ( in_the_loop() ) {
+				$pod_name = $post->post_type;
+			} else {
+				// Outside the loop in a taxonomy, we want the term
+				if ( is_tax() ) {
+					global $wp_query;
+					$obj      = $wp_query->get_queried_object();
+					$pod_name = $obj->slug;
+					$pod_item = $obj->term_id;
+				}
+			}
```

Comment:
Just wondering whether this is always correct.
What about Users or Post Type Archives (non-term archives).
Is that something we should look into? Or isn't that supported at all?
Might be better to tackle this separately from this PR.

Reply:
This appears to be always correct for our supported use cases here.